client/lib/domains/registration/availability-messages.js

```diff
@@ -146,10 +149,20 @@ function getAvailabilityNotice( domain, error, site ) {
 			break;
 		case domainAvailability.MAINTENANCE:
 			if ( tld ) {
+				let maintenanceEnd = translate( 'shortly', {
+					comment: 'If a specific maintenance end time is unavailable, we will show this instead.',
+				} );
+				if ( maintenanceEndTime ) {
+					maintenanceEnd = moment.unix( maintenanceEndTime ).fromNow();
+				}
+
 				message = translate(
-					'Domains ending with {{strong}}.%(tld)s{{/strong}} are undergoing maintenance. Please check back shortly.',
+					'Domains ending with {{strong}}.%(tld)s{{/strong}} are undergoing maintenance. Please check back %(maintenanceEnd)s.',
```

Comment:
I know it's how the original copy was, but maybe we could add ` or check a different extension.` at the end? To give the user a hint (in case it was not clear) that other TLDs _are_ working.